src/gui/screens.py

```diff
@@ -63,15 +68,19 @@ def cb(obj, event):
                         Pin.reset_counter(alert)
                         callback()
                     else:
-                        instruct_label.set_text("Wrong pin: %d/%d" % (Pin.counter, Pin.ATTEMPTS_MAX))
+                        instruct_label.set_text("#f07070 Wrong pin: %d/%d #" % (Pin.counter, Pin.ATTEMPTS_MAX))
                         if Pin.counter <= 0:
                             Factory_settings.restore(alert)
                             Secret.generate_secret()
                             alert("Security","Device has been factory reset!")
                             first_time_usage = True
+                            title_lbl.set_text(first_time_title)
+                            instruct_label.set_text(instruct_txt)
+                            return
```

Comment:
after factory reset the screen should be clean of any anti-phishing words

Reply:
Thanks! Fixed :)